h/templates/layouts/base.html

```diff
@@ -10,7 +10,13 @@
       {% endfor -%}
     {% endblock %}
 
-    <title>{% block title %}Hypothesis{% endblock %}</title>
+    <title>
+      {%- block title -%}
+        {%- block page_title %}{% endblock -%}
+        {%- if self.page_title() %} | {% endif -%}
```

Comment:
Or rather, if you simply remove the block definition above this line, then you can test with an `if` whether the child defined it.